source/db/zh-CN-projects.json

```diff
@@ -1287,32 +1289,7 @@
   },
   {
     "development_stage": "released",
-    "description": "私密的 XMPP 服务，运行在 DuckDuckGo 社区平台。&mdash;包含使用说明。",
```

Comment:
This definitely should be gone, but should be done in a separate commit. https://www.freshconsulting.com/atomic-commits/

Not sure if you did this on purpose.